### PR TITLE
Wfh/google docs update

### PR DESCRIPTION
**Pull request by hinthornw**
- Add gemini references
- Fix the notebook (ultra isn't generally available; also gemini will randomly filter out responses, so added a fallback)

### CODE REVIEW

File: docs/docs/integrations/platforms/google.mdx [hunk 1]
```diff
@@ -2,41 +2,56 @@
...
 
-Access `PaLM` LLMs like `text-bison` and `code-bison` via `Google Vertex AI`.
+### ChatGoogleGenerativeAI
 
-We need to install `google-cloud-aiplatform` python package.
+Access `Gemini` models such as `gemini-pro` and `gemini-pro-vision` through the `ChatGoogleGenerativeAI` class.
 
 ```bash
-pip install google-cloud-aiplatform
+pip install -U langchain-google-genai
```
Comment (jamesbraza) [inline docs/docs/integrations/platforms/google.mdx:12]: @hinthornw would you mind adding the `url` parameter to the package metadata?

That way https://pypi.org/project/langchain-google-genai/ can link back to either LangChain repo root or the package's subfolder

Comment (hinthornw) [reply docs/docs/integrations/platforms/google.mdx:12]: Will do! In a separate PR though

Comment (jamesbraza) [reply docs/docs/integrations/platforms/google.mdx:12]: Thanks! Sorry for lurking, wasn't sure a better way to communicate this to you

Comment (hinthornw) [reply docs/docs/integrations/platforms/google.mdx:12]: Always appreciate the comments and requests!

**Pull request merged**